packages/opentelemetry-api/src/platform/node/IdGenerator.ts

```diff
@@ -14,23 +14,15 @@
  * limitations under the License.
  */
 
-import * as crypto from 'crypto';
-
-const SPAN_ID_BYTES = 8;
-const TRACE_ID_BYTES = 16;
-
 /**
- * Returns a random 16-byte trace ID formatted/encoded as a 32 lowercase hex
- * characters corresponding to 128 bits.
+ * IdGenerator provides an interface for generating Trace Id and Span Id
+ *
+ * The prototype IdGenerator generates random 16-byte trace ID and 8-byte span ID
  */
-export function randomTraceId(): string {
-  return crypto.randomBytes(TRACE_ID_BYTES).toString('hex');
-}
 
-/**
- * Returns a random 8-byte span ID formatted/encoded as a 16 lowercase hex
- * characters corresponding to 64 bits.
- */
-export function randomSpanId(): string {
-  return crypto.randomBytes(SPAN_ID_BYTES).toString('hex');
+export interface IdGenerator {
```

Comment:
Thank you for your comment and sorry about that, I am really not sure where should this interface be added. Because current id.ts is under platform folder, I simply followed it and added interface to api/.../platform folder. If there is any instruction for where it should be placed would be really helpful :)

Reply:
Oh, you mentioned I should move interface outside of platform. Did you mean I should move it from ".../platform/node" to ".../platform" because both web and node version are using the same interface, or elsewhere? @dyladan

Reply:
I would create a new place for it outside of `platform`